src/pydartdiags/matplots/matplots.py

```diff
@@ -265,7 +289,7 @@ def plot_rank_histogram(obs_seq, levels, type, ens_size):
             ax2.set_xlabel("Observation Rank (among ensemble members)")
             ax2.set_ylabel("Count")
 
-        fig.suptitle(f"{type} at Level {level}", fontsize=14)
+        fig.suptitle(f"{type} at Level {level} {unit}", fontsize=14)
```

Comment:
unit here for an ocean case would be "Depth (m)"
so title is awkward: example XBT_TEMPERATURE at Level 25.0 Depth (m)
I suggest redefining unit as just "m" (similar to hPa logic)